src/DotNetLightning.Core/Channel/ChannelError.fs

```diff
@@ -145,6 +147,15 @@ and InvalidAcceptChannelError = {
         Errors = e
     }
     
+and InvalidMonoHopUnidirectionalPaymentError = {
+    Msg: MonoHopUnidirectionalPayment
+    Errors: string list
+}
+    with
+    static member Create msg e = {
```

Comment:
hey let's remove this Create method, as the record can be instantiated easily anyway without it

Reply:
I can do. All the other invalid msg types have it though, though they all seem equally pointless. Is it important to be consistent here?

Reply:
good point; let's remove it here and I'll prepare a PR to remove them upstream

Reply:
Done.